doc/development.md

```diff
@@ -14,6 +14,12 @@ Enable oVirt packages for Fedora:
     sudo dnf copr enable -y nsoffer/ioprocess-preview
     sudo dnf copr enable -y nsoffer/ovirt-imageio-preview
 
+Enable
+[virt-preview](https://copr.fedorainfracloud.org/coprs/g/virtmaint-sig/virt-preview/)
+package to obtain latest qemu and libvirt versions (and update if needed):
```

Comment:
Enable [virt-preview](...) repository ...

What do you mean by "(and update if needed)"?

Reply:
Well, after adding the **repository** (I'll fix it!), one may need to do `dnf update`, I doubted if it was worth mentioning it explicitly, or if I could just assume people would know they have to update afterwards, so I ended up somewhere in between.

If it is not clear, I will re-phrase it or explicitly write the command.

Reply:
If we need to do "dnf update", it should be a documented step. But I don't remember that this is neededl.

Reply:
Well, adding the repository alone does not update qemu and libvirt versions if it is required. I had to run `sudo dnf update` myself if I remember it correctly.

Reply:
But these steps are done before installing vdsm - when you install vdsm it will pull the right version of qemu and libvirt from the virt preview repo.

Testing this require starting with a clean host and running the steps one by one.